index.html

```diff
@@ -38,42 +38,34 @@ <h2 class="second-header">I am software developer</h2>
         </p>
```

Comment:
[Optional] Please fix the margin around the  hamburger menu 
![image](https://user-images.githubusercontent.com/74744787/159040020-053d86ca-2a30-49a0-a973-5c31f2ea73bc.png)